main/src/ca/uwaterloo/flix/language/phase/MonoTyper.scala

```diff
@@ -39,7 +39,8 @@ object MonoTyper {
     val cs = def0.cparams.map(visitFormalParam)
     val fs = def0.fparams.map(visitFormalParam)
     val s = visitStmt(def0.stmt)
-    val tpe = visitType(def0.tpe)
+    val tpe0 = visitType(def0.tpe)
+    val tpe = MonoType.Arrow(fs.map(_.tpe), tpe0)
```

Comment:
So in `ReducedAst`, `tpe` means "return type" and in `MonoTypedAst` it means "function arrow type". Are there plans to make this consistent?

Reply:
We should make it consistent.